db/seeds.rb

```diff
@@ -7,3 +7,4 @@
 # it should be safe to run multiple times in the same environment, and to remove
 # the relevant ENV vars after it has run at least once.
 RolesConversion.new.ncr_budget_approvers
+RolesConversion.new.gsa18f_approvers
```

Comment:
ok, that's a good start! I am still *really* hoping to find a way to not run seeds, but baby steps :baby: :chart_with_upwards_trend: